.github/workflows/pr-version-comment.yml

```
@@ -0,0 +1,29 @@
+name: Comment App Version on Merged PR
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  comment-version:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'main'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out main
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Read version.txt
+        id: get_version
+        run: echo "version=$(cat version.txt)" >> $GITHUB_OUTPUT
+
+      - name: Comment on PR
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            PR merged to `main`, targeting: `${{ steps.get_version.outputs.version }}`
```

A tiny wording suggestion:

```suggestion
            PR merged to `main`, targeting version `${{ steps.get_version.outputs.version }}`
```

---

Fixed, also added an emoji just for fun 👀  I imagine the phrasing will be adjusted as the team continues to try this out.

---

Hmm, now thinking about this … this is a firefox-ios `version.txt` … that's not used for focus-ios (and may also be irrelevant for automation etc. PRs) — I'm wondering whether to add `on: paths` to the trigger, and only list file patterns that are relevant to be in the diff to trigger this FxiOS version number?